react/Header/Header.less

```diff
@@ -33,6 +27,7 @@
   align-items: center;
   flex: 1 0 auto;
   padding: 0 (@grid-gutter-width / 2);
+  max-width: @grid-container-width + @grid-gutter-width;
```

Comment:
Obviously the end result works, but why do we only add a single gutter width as opposed to two?